### DISCUSSION TITLE
UBL commands gcode

### DISCUSSION THREAD

**Discussion by 5P4P** [supermerill/SuperSlicer#1810] [Q&A] [answered]
Hi,

I have been trying out this slicer and am loving it sofar. Im using Ender3 V2 with jyers firmware BLTouch.
One thing i cant find is where can i add lines to the start of gcode. Now i need to change it every time after export.
After **G28 ; home all axis** I want to add 
**G29 L0 ; Load UBL
G29 J ; Quick 3-point level**

Thanks

**Comment by supermerill** [comment, marked as answer]
Click a magnifying glass and enter "start gcode", then select the first entry.

or go to printer settings -> custom gcode -> start gcode

**Reply by 5P4P** [reply]
It does not find anything. and this is what i have under printer settings

![image](https://user-images.githubusercontent.com/93838745/140792093-98f8b53a-dddf-4d8e-a966-55c159a9dd2e.png)
![image](https://user-images.githubusercontent.com/93838745/140792252-e6dccc3e-adf9-4729-847c-cde4ca68c957.png)

**Reply by supermerill** [reply]
Oh. you have to select the expert mode to be able to see the custom gcode field.
I forgot that.
Maybe I should do something to let the search search for everything...

**Reply by 5P4P** [reply]
Ahh :D OK i see it now. Thanks!
